crates/bindgen-core/src/lib.rs

```diff
@@ -8,6 +8,12 @@ mod ns;
 
 pub use ns::Ns;
 
+/// Helper for getting the version of the bindgen.
+/// If CARGO_VERSION_INFO is set, use it, otherwise use CARGO_PKG_VERSION.
+pub fn version() -> &'static str {
+    option_env!("CARGO_VERSION_INFO").unwrap_or(env!("CARGO_PKG_VERSION"))
```

Comment:
If this is moved here the `build.rs` is going to need to move in tandem since that's what defines the `CARGO_VERSION_INFO` env var

Reply:
That's a really good point!